### PR TITLE
Album details UI fix

### PR DESCRIPTION
**Pull request by rochakjain361**
Fixes https://github.com/navidrome/navidrome/issues/910
Before:
112582542-c3caac00-8e1c-11eb-94c5-a7aec7eebc11
Now: 
Screenshot from 2021-03-29 01-11-15

### CODE REVIEW

**Comment by rochakjain361** [comment]
@deluan @jvoisin Can you review this code? And mention any changes if required.

File: ui/src/album/AlbumListView.js [hunk 1]
```diff
@@ -34,24 +40,54 @@ const useStyles = makeStyles({
...
   contextMenu: {
     visibility: 'hidden',
   },
 })
 
 const AlbumDetails = (props) => {
+  const classes = useStyles()
+  const translate = useTranslate()
+  const { record } = props
+  console.log(props)
```
Comment (deluan) [inline ui/src/album/AlbumListView.js:55]: Please remove

**Review by deluan — CHANGES_REQUESTED** [review]
Thanks for PR. There is a big gap between the Album row and the first field in the details:
Screen Shot 2021-03-29 at 10 00 20 PM

This does not happen with the Song detail:
Screen Shot 2021-03-29 at 10 00 32 PM

**Comment by rochakjain361** [comment]
@deluan I've made the requested changes to the pull request, can you please review it.

Screenshot from 2021-03-31 03-24-48

Screenshot from 2021-03-31 03-25-04

Also, I have a suggestion before closing this issue, we should display the artist's name in the album's detail view. While importing albums, the album's artist is taken as `Various Artists` many times and the real artist's name was in the `Artist` field. So it will be better if we display the Artist's name as well. 

Screenshot from 2021-03-31 04-23-13

**Comment by deluan** [comment]
Thanks for the PR. LGTM, merging.

>While importing albums, the album's artist is taken as Various Artists many times and the real artist's name was in the Artist field

There is no "real artist" for a "various artists" album, only for the songs contained in it. Ex: "Best of 2017", various artists (it is a compilation): who is the "real artist"?

We will probably show all artists in that album when we get multiple artists (#238) in place.

**Pull request merged**